src/components/Calculator.js

```diff
@@ -1,61 +1,56 @@
-import React, { PureComponent } from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] I would like to recommend you use an `arrow `function instead of a regular function. :+1: 
```
const Calculator = () => (
--------
)
```